.travis.yml

```diff
@@ -2,18 +2,24 @@ language: scala
 sudo: false
 matrix:
   include:
-    - scala: 2.10.6
-      script: umask 0022 && ./sbt ++$TRAVIS_SCALA_VERSION clean test mimaReportBinaryIssues
-
     - scala: 2.11.8
+      jdk: oraclejdk8
       script: umask 0022 && ./sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport mimaReportBinaryIssues
       after_success:
         - bash <(curl -s https://codecov.io/bash)
+#    - scala: 2.12.1
```

Comment:
I think we dont need to run coverage (as we're doing this for 2.11) and mima (as this will be the first 2.12 build).